docker/py3.7/Dockerfile

```diff
@@ -1,19 +1,19 @@
-# Build as renku/singleuser
 # Run with the RenkuKubeSpawner via JupyterHub
-
-ARG JUPYTERHUB_VERSION=1.2
-ARG BASE_IMAGE=jupyterhub/singleuser:$JUPYTERHUB_VERSION
+ARG BASE_IMAGE=jupyter/base-notebook:8fd7ada2a374
```

Comment:
maybe you already said, but why the one with the hash and not any of the particular tags? (also it was updated again 2 days ago so we could bump this); https://hub.docker.com/r/jupyter/base-notebook/tags

Reply:
which tag would you choose? They are all the same image btw.

Reply:
Probably this one `jupyter/base-notebook:hub-1.1.0` because we care about the version of jupyterhub to be compatible with whatever we have in `notebooks`?

Reply:
yeah but that means that everything else can change because they can push a new version that has all other libraries changed _except_ JH and we wouldn't know. If we use a specific hash we guarantee that it's always going to be exactly the same image.

Reply:
If you want, we can run `pip freeze` in the base image and store the relevant versions of dependencies that we care about somewhere in a comment, for example

Reply:
ohhh, no that's a good point; now I understand why they have all those different tags (and also that we could but shouldn't do a similar thing). we can keep the hash!